package.json

```diff
@@ -25,7 +25,7 @@
   "version": "0.26.0",
   "private": true,
   "dependencies": {
-    "@hathor/wallet-lib": "^0.46.1",
+    "@hathor/wallet-lib": "^1.0.0-rc5",
```

Comment:
We still require the wallet desktop utils pr

Reply:
We can bump this to rc8 already :rocket: